### PR TITLE
[MCKIN-18911] Review fix for participants count on company detail page

### PR DESCRIPTION
**Pull request by sohaibaslam**
Added filter for excluding relevant admin users for a course.

Issue: https://edx-wiki.atlassian.net/browse/MCKIN-18911

### CODE REVIEW

File: edx_solutions_organizations/views.py [hunk 1]
```diff
@@ -318,25 +318,16 @@ def courses(self, request, pk):  # pylint: disable=W0613
...
-            .filter(user__organizations=organization, is_active=True)\
-            .exclude(
-                user_id__in=CourseAccessRole.objects.filter(
-                    course_id__in=organization_courses, role__in=roles_to_exclude
-                ).values_list('user_id', flat=True)
-            ).values_list('course_id', 'user_id')
+        enrollment_qs = CourseEnrollment.objects.filter(user__organizations=organization, is_active=True).annotate(
+            user_with_role=Exists(
+                CourseAccessRole.objects.filter(
+                    course_id=OuterRef('course_id'), role__in=roles_to_exclude, user_id=OuterRef('user_id'))
```
Comment (nasirhjafri) [inline edx_solutions_organizations/views.py]: We considered using `OuterRef` in past. But we are on `Django==1.8.19` and it doesn't have `OuterRef`.

**Review by nasirhjafri — CHANGES_REQUESTED** [review]
If this fix is intended for Ironwood, please check which version of Django we are using in Ironwood.

**Comment by sohaibaslam** [comment]
> If this fix is intended for Ironwood, please check which version of Django we are using in Ironwood.

We are using django==1.11.29 in Ironwood.

**Comment by moeez96** [comment]
@sohaibaslam Please change the base of your branch to `master-ginko`

**Review by nasirhjafri — APPROVED** [review]

**Review by mudassir-hafeez — COMMENTED** [review]
please update version in setup.py

**Pull request merged**